drv/ArrayFIFOQueue.drv

```diff
@@ -123,11 +140,11 @@ public class ARRAY_FIFO_QUEUE KEY_GENERIC implements PRIORITY_QUEUE KEY_GENERIC,
 		length = newLength;
 	}
 
-	private final void expand() {
+	protected void expand() {
```

Comment:
Not sure if this should be overrideable. Its a rather internal functionality.

Reply:
True, but I have seen a lot of cases where this function or the shrink function were the target of overrides.
So instead of overriding 3 functions that all call this exact function (which they want to modify in the first place) it is overriding the function directly. resize is still private.

Reply:
Something @vigna should decide IMO